test/e2e/framework/.import-restrictions

```diff
@@ -195,6 +195,7 @@ rules:
       - k8s.io/kubernetes/pkg/scheduler/listers
       - k8s.io/kubernetes/pkg/scheduler/metrics
       - k8s.io/kubernetes/pkg/scheduler/nodeinfo
+      - k8s.io/kubernetes/pkg/scheduler/testing
```

Comment:
Without this, I got the following error:

```
E0128 22:35:58.441201   79078 main.go:45] Error: Failed executing generator: some packages had errors:
errors in package "k8s.io/kubernetes/test/e2e/framework/providers/kubemark":
the following imports did not match any allowed prefix:
  k8s.io/kubernetes/pkg/scheduler/testing
```

Reply:
can you tell what is the chain that introduces this dependency to see if we can avoid it?

Reply:
to be honest, I don't have a clear clue... the error is also confusing. Will dig more.

Reply:
Seems other people got confused about the error:

https://github.com/kubernetes/kubernetes/blob/0d598baa4aed2ccbc7c658100b45a0ab993044a4/test/e2e/framework/.import-restrictions#L244

@alculquicondor instead of digging further, a simple workaround is to eliminate the usage of sched testing package in pkg/scheduler/internal/queue/scheduling_queue_test.go:

```go
st.MakePod().Name("p").Obj()
```

WDYT?

Reply:
It looks like the bigger problem is that `test/e2e/framework` depends on  `pkg/scheduler/internal/queue`, which is probably a longstanding issue. So now I'm not too concerned.

Another problem is that import boss cares about `_test.go` dependencies :/

Reply:
done.